manifest.yml

```diff
@@ -1,11 +1,13 @@
 name: netlify-plugin-chromium
 inputs:
-  - name: packageManager
-    description: Package manager to install Chromium with; npm or yarn.
-    default: npm
   - name: setChromePathInEnv
     description: If true, sets value of environment variable CHROME_PATH to location of local copy of Chromium binaries. This change is required by many tools relying on Chromium to be able to find it and launch it successfully (such as Lighthouse).
     default: true
+  # Not used anymore, but kept for backward compatibility
+  - name: packageManager
+    description: Package manager to install Chromium with; npm or yarn.
+    default: npm
+  # Not used anymore, but kept for backward compatibility
```

Comment:
Removing those from `manifest.yml` would be a breaking change, since it would make any build using them fail.